src/libasr/codegen/asr_to_c.cpp

```diff
@@ -806,6 +818,11 @@ R"(
     }
 
     void visit_Module(const ASR::Module_t &x) {
+        if (startswith(x.m_name, "lfortran_intrinsic_")) {
+            intrinsic_module = true;
+        } else {
+            intrinsic_module = false;
+        }
```

Comment:
https://github.com/lfortran/lfortran/pull/1747#discussion_r1215537517

Reply:
We can fix this in a separate PR at all places (this is not the only occurrence).